src/components/preview/preview.vue

```diff
@@ -35,6 +37,27 @@
         type: String,
         required: true
       }
+    },
+    data () {
+      return {
+        criteria: {}
+      }
+    },
+    computed: {
+      criteriaIsMet () {
+        if (Object.keys(this.criteria).length > 0) {
```

Comment:
`Object.keys(this.criteria).length` returns truthy so no need to check if it's larger than 0